client/src/components/FeedbackPage/FeedbackTestimonialsSection/FeedbackTestimonialsSection.tsx

```diff
@@ -52,29 +48,33 @@ const FeedbackTestimonialsSection: FunctionComponent = () => {
           <h2>Testimonials</h2>
           {data.length > 0 ? (
             <RequestDataWrapper data={data} loading={loading} error={error}>
-              {testimonialsDataArray?.map(({ id, content }, index) => (
-                <Accordion key={id} title={testimonialsConfigArray[index].title} index={index}>
-                  {content.list
+              {testimonialsDataArray?.map(({ id, list }, index) => (
+                <Accordion key={id} title={testimonialsConfig[id].title} index={index}>
+                  {list
                     .filter(item => item.message)
-                    .map(({ message, name, rating, timestamp }, i) => {
+                    .map(({ name, message, rating, timestamp }, i) => {
                       const { day, year, time } = dateFormatter(timestamp, true);
 
                       return (
                         <S.FeedbackTestimonial key={`testimonial-${i + 1}`} themeType={theme}>
                           <Flex directionColumn alignFlexStart>
                             <S.FeedbackTestimonialTop>
                               {rating && (
-                                <Flex justifyFlexStart>
-                                  {[...Array(5)].map((_, j) => (
-                                    <S.FeedbackTestimonialStarWrapper
-                                      key={`star-${j + 1}`}
-                                      active={rating >= j + 1}
-                                      themeType={theme}
-                                    >
-                                      <IconStar />
-                                    </S.FeedbackTestimonialStarWrapper>
-                                  ))}
-                                </Flex>
+                                <StarRating
+                                  theme={theme}
+                                  inputConfig={{
+                                    type: 'radio',
+                                    options: {
+                                      'rating-1': 1,
+                                      'rating-2': 2,
+                                      'rating-3': 3,
+                                      'rating-4': 4,
+                                      'rating-5': 5
+                                    }
```

Comment:
1. looks like options for star rating component are always the same. can you please move them to separate variable and import in feedback form and testimonials component?
2. `readOnlyRating` can be renamed to `defaultValue` (it's more common and popular name for this prop)
https://reactjs.org/docs/uncontrolled-components.html#default-values